sdc/rabbit/consumers.py

```diff
@@ -564,13 +562,11 @@ def on_message(self, unused_channel, basic_deliver, properties, body):
                                     requeue=True,
                                     tx_id=tx_id)
 
-        except RetryableError as e:
+        except RetryableError:
             self.nack_message(basic_deliver.delivery_tag, tx_id=tx_id)
-            logger.error("Failed to process",
-                         action="nack",
-                         exception=str(e),
-                         tx_id=tx_id)
-
+            logger.exception("Failed to process",
```

Comment:
Is the removal of `e` this a log pollution avoidance thing?

Reply:
It's just less lines of code.  If you do logger.exception then you get the stack trace for free.  Then you can have the log line be a bit smaller with less fields defined.
It's just a tidier way to do the same thing.